Dockerfile

```diff
@@ -6,8 +6,12 @@ COPY . /usr/src/app
 
 RUN apt-get --yes update \
     && apt-get --yes upgrade \
-    && pip3 install pipenv \
-    && pipenv install --system --deploy --dev
+    && pip3 install pipenv
+
+RUN pip3 install mysqlclient
+RUN pipenv install --system --deploy --dev
 
 EXPOSE 8000
-CMD ["python3", "manage.py", "runserver", "0.0.0.0:8000"]
+
+RUN sleep 10
```

Comment:
I don't understand this change. Why run `sleep 10` inside the container?

Reply:
Was a testing statement that doesn't hurt anything, but should be removed.

Reply:
@nlMeminger No worries. Let's remove it before this PR merges though, it will be annoying to wait an extra ten seconds for the container to build.